bundle/config/generate/pipeline.go

```diff
@@ -0,0 +1,14 @@
+package generate
+
+import (
+	"github.com/databricks/cli/libs/dyn"
+	"github.com/databricks/cli/libs/dyn/yamlsaver"
+	"github.com/databricks/databricks-sdk-go/service/pipelines"
+)
+
+var pipelineOrder = yamlsaver.NewOrder([]string{"name", "new_cluster", "existing_cluster_id", "libraries"})
+
+func ConvertPipelineToValue(pipeline *pipelines.PipelineSpec) (dyn.Value, error) {
+	value := make(map[string]dyn.Value)
+	return yamlsaver.ConvertToMapValue(pipeline, pipelineOrder, []string{"format", "id", "storage", "edition", "channel"}, value)
```

Comment:
The channel field is relevant; customers can opt into the preview channel.

Could you include a bullet list for each field saying why we ignore it? For example the storage field; if we change it on apply, I believe this means we force recreation of the pipeline instead of maintaining it as is.